README.md

```diff
@@ -79,7 +85,11 @@ For example:
 
 ## Multiple Suites
 
-Coverage from multiple suites can be sent to Code Climate by aggregating each suite's results into one final report. To accomplish this, follow the instructions for [Parallel Tests](#parallel-tests) above. If the suites produce reports in the same location on disk, you can omit any steps that involve uploading and downloading partial reports.
+Coverage from multiple suites can be sent to Code Climate by aggregating each
+suite's results into one final report. To accomplish this, follow the
+example for [Parallel Tests](#parallel-tests) above. If the suites produce
+reports in the same location on disk, you can omit any steps that involve
+uploading and downloading partial reports.
```

Comment:
I didn't say it at the time, but I find the paragraph potentially confusing because it references other instructions then vaguely tells you to modify them. WDYT about "duplicating" the stuff above?

> 1. After each test suite, run:
>
>    ```
>    ./cc-test-reporter format-coverage --output coverage/codeclimate.$SUITE.json
>    ```
>
> 1. After all test suites, run:
>
>    ```
>    ./cc-test-reporter sum-coverage coverage/codeclimate.*.json | \
>      ./cc-test-reporter upload-coverage
>    ```